bin/pubber

```diff
@@ -23,4 +23,35 @@ $ROOT_DIR/pubber/bin/build > /dev/null
 
 echo Running tools version `(cd $ROOT_DIR; git describe)`
 
-$ROOT_DIR/pubber/bin/run $project_id $site_path $device_id $serial_no
+if [[ "${#}" -gt "0" ]]; then   
```

Comment:
I would flip the if condition so the short branch is first, rather than after the else (just makes it easier to read)